Storage Queues now support infinite message TTL

theunrepentantgeek opened the pull request:
Introduced in api-version 2017-07-27, messages can now opt-in to infinite TTL by specifying an expiration of -1 seconds. The default remains at 7 days. 

See https://docs.microsoft.com/en-us/rest/api/storageservices/versioning-for-the-azure-storage-services

azurecla commented:
Hi __@nk773__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (neerajkh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
Please submit freshness updates through the private repo.

theunrepentantgeek committed:
Storage Queues now support infinite message TTL 

Introduced in api-version 2017-07-27, messages can now opt-in to infinite TTL by specifying an expiration of -1 seconds. The default remains at 7 days. 

See https://docs.microsoft.com/en-us/rest/api/storageservices/versioning-for-the-azure-storage-services
